src/lib/containers/MarkdownSynapse.tsx

```diff
@@ -575,33 +576,35 @@ export default class MarkdownSynapse extends React.Component<
     if (alignLowerCase === 'right') {
       buttonClasses += 'floatright '
     }
-    if (highlight === 'true') {
-      buttonClasses += ' SRC-primary-button '
-    } else {
-      buttonClasses += 'SRC-light-button '
-    }
+    const buttonVariant =
+      highlight === 'true' ? 'primary' : 'light-primary-base'
```

Comment:
may want to check for truthy value instead (which I think would include the string 'true')